oxenc/bt_producer.h

```diff
@@ -341,6 +340,20 @@ class bt_list_producer {
         append_intermediate_ends();
     }
 
+    /// Appends an input list container as a sublist.  Equivalenet to append_list(tuple).  Note that
+    /// this is not equivalent to the iterator pair overload above: that appends to the current
```

Comment:
It almost feels like we should do like python with `extend` vs `append`

Reply:
Yeah, it's awkward, but I don't want to break anything that might be using the existing api.